dist/README.md

```diff
@@ -0,0 +1,27 @@
+# Leaderboard-list-app
+Demonstrate the use of Callbacks, Promises, ES6, Webpack and Modular JS
```

Comment:
- Please provide a better description of your App.
- **NB: These Requirements because your ReadMe file is the first contact between your project and maybe a recruiter. Having a professional Readme can lead you to your first job contract**

Reply:
Thanks @OLIPLICHE for the comments.Let me Implement them